modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowNotifications.testcase

```diff
@@ -112,12 +112,48 @@ definition {
 	}
 
 	@description = "Verify that the user can open new user details by notifications"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOpenNewUserDetailsByNotifications {
+		property test.name.skip.portal.instance = "WorkflowNotifications#CanOpenNewUserDetailsByNotifications";
 
-		// TODO LRQA-70881 CanOpenNewUserDetailsByNotifications pending implementation
+		task ("Configure Workflow to Single Approver for User") {
```

Comment:
Hey @sirfabios  can you change the task description to something like:  "When bla bla"